libcloud/compute/drivers/gce.py

```diff
@@ -5158,7 +5166,8 @@ def _create_node_req(self, name, size, image, location, network=None,
             if not ex_disk_type:
                 ex_disk_type = 'pd-standard'
             if not hasattr(ex_disk_type, 'name'):
-                ex_disk_type = self.ex_get_disktype(ex_disk_type)
+                ex_disk_type = self.ex_get_disktype(ex_disk_type,
```

Comment:
When I looked at the code, it seems passing in None for zone will cause the disk type not to be detected so my code didn't work without that.